DataManagementSystem/DB/FileCatalogComponents/DirectoryLevelTree.py

```diff
@@ -253,12 +255,6 @@ def getDirectoryPaths(self, dirIDList):
       return S_OK({})
 
     dirListString = ','.join([str(d) for d in dirs])
-
-    dirListString = self.db._escapeString(dirListString)
```

Comment:
Why do you remove this part ? You have no guarantee that it is indeed integers that you have there

Reply:
We escape user input strings, e.g. lfns that come through the client interface. Those are numerical IDs internal to the catalog, no need to escape them

Reply:
oh ok, sorry, I thought this method was exposed outside, my bad